src/magic_folder/test/cli/test_magic_folder.py

```diff
@@ -264,17 +276,38 @@ def _clean(ign):
 
     def init_magicfolder(self, client_num, upload_dircap, collective_dircap, local_magic_dir, clock):
         dbfile = abspath_expanduser_unicode(u"magicfolder_default.sqlite", base=self.get_clientdir(i=client_num))
+        try:
+            db=get_magicfolderdb(dbfile, create_version=(SCHEMA_v1, 1))
```

Comment:
Elsewhere the return value of `get_magicfolderdb` is checked against `None`, as if errors are signaled that way.  I wonder which it is.